src/test/java/com/adobe/target/edge/client/ondevice/DefaultRuleLoaderTest.java

```diff
@@ -353,4 +354,20 @@ void testGetLocationWithoutProperty() {
     assertEquals("https://assets.adobetarget.com/client123/production/v1/rules.json", artifactUrl);
     defaultRuleLoader.stop();
   }
+
+  @Test
+  void testGetArtifactoryWithTelemetry() {
```

Comment:
Nitpick - please rename to `testGetArtifactWithTelemetry`.  I think I accidentally included Artifactory in the jira ticket a while ago, but Artifactory is something unrelated to this.